### ISSUE TITLE
Error starting validator on Windows 10 

### ISSUE THREAD

**Issue by madhancr** [prysmaticlabs/prysm#5538]
# 🐞 Bug Report

### Description

 A clear and concise description of the problem...

### Has this worked before in a previous version?

 Yes, the previous version in which this bug was not present was: ....

## 🔥 Error

<pre><code>

</code></pre>

## 🌍  Your Environment

**Operating System:**

<pre>
  <code>

  </code>
</pre>

**What version of Prysm are you running? (Which release)**

<pre>
  <code>

  </code>
</pre>

**Anything else relevant (validator index / public key)?**

**Comment by madhancr** [comment]
Error:
<!--------
docker run -it -v $HOME/prysm/validator:/data --network="host" gcr.io/prysmaticlabs/prysm/validator:latest --beacon-rpc-provider=127.0.0.1:4000 --keymanager=keystore --keymanageropts='{"path":"/data","passphrase":"changeme"}'
2020/04/20 04:45:27 maxprocs: Leaving GOMAXPROCS=2: CPU quota undefined
[2020-04-20 04:45:27]  WARN flags: Using default mainnet config
[2020-04-20 04:45:27]  WARN flags: Disabled validator proposal slashing protection.
[2020-04-20 04:45:27]  WARN flags: Disabled validator attestation slashing protection.
The keystore key manager generates keys and stores them in a local encrypted store.  The options are:
  - path This is the filesystem path to where keys will be stored.  Defaults to the user's home directory if not supplied
  - passphrase This is the passphrase used to encrypt keys.  Will be asked for if not supplied
A sample set of options are:
  {
    "path":   "/home/me/keys", // Store the keys in '/home/me/keys'
    "passphrase": "secret"     // Use the passphrase 'secret' to encrypt and decrypt keys
  }
[2020-04-20 04:45:27] ERROR main: invalid character 'p' looking for beginning of object key string
--------------------!>
Env: Windows 10 Pro (command prompt)
Topaz Testnet 

I was able to install docker and run the beacon node and generate the keys, but validator command fails.

**Comment by prestonvanloon** [comment]
Hi @madhancr, what is the issue here? Please provide as much details as possible about your situation so we can best triage your issue.

**Comment by madhancr** [comment]
sorry forgot to paste the error:

docker run -it -v $HOME/prysm/validator:/data --network="host" gcr.io/prysmaticlabs/prysm/validator:latest --beacon-rpc-provider=127.0.0.1:4000 --keymanager=keystore --keymanageropts='{"path":"/data","passphrase":"changeme"}'
2020/04/20 04:45:27 maxprocs: Leaving GOMAXPROCS=2: CPU quota undefined
[2020-04-20 04:45:27]  WARN flags: Using default mainnet config
[2020-04-20 04:45:27]  WARN flags: Disabled validator proposal slashing protection.
[2020-04-20 04:45:27]  WARN flags: Disabled validator attestation slashing protection.
The keystore key manager generates keys and stores them in a local encrypted store.  The options are:
  - path This is the filesystem path to where keys will be stored.  Defaults to the user's home directory if not supplied
  - passphrase This is the passphrase used to encrypt keys.  Will be asked for if not supplied
A sample set of options are:
  {
    "path":   "/home/me/keys", // Store the keys in '/home/me/keys'
    "passphrase": "secret"     // Use the passphrase 'secret' to encrypt and decrypt keys
  }
[2020-04-20 04:45:27] ERROR main: invalid character 'p' looking for beginning of object key string

**Comment by shayzluf** [comment]
Hi @madhancr ,
Thank you for the provided details . Making windows UX better is on our priorities before mainnet release. till then can you please try to follow this guide https://kb.beaconcha.in/tutorial-eth2-multiclient/docker-beaconnode-and-validator
the windows command for running validator there is `docker run -it -v c:/prysm:/data --network="host" gcr.io/prysmaticlabs/prysm/validator:latest --beacon-rpc-provider=127.0.0.1:4000 --keystore-path=/data --datadir=/data --password=yourPassword`

**Comment by madhancr** [comment]
thanks, I was able to get past that error. But running into another issue.

1) Got beacon note running and fully synced
2) Generated keys and pasted that into UI in https://prylabs.net/participate - Step 5

ISSUE:
After submitting transaction data, metamask gave transaction confirmation but the prysm UI was just spinning (https://prylabs.net/participate). When I refreshed the page after a few mins, I can not to go past Step 2

Validator is Waiting 
[2020-04-27 22:46:51]  INFO validator: Waiting for deposit to be seen pubKey=0xaf59f5318095 status=UNKNOWN_STATUS
[2020-04-27 22:47:03]  INFO validator: Waiting for deposit to be seen pubKey=0xaf59f5318095 status=UNKNOWN_STATUS
